search/includes/classes/commands/class-healthcommand.php

```diff
@@ -262,20 +262,26 @@ private function render_results( array $results ) {
 	 * [--silent]
 	 * : Optional silences all non-error output except for the final results
 	 *
+	 * [--force-parallel-execution]
+	 * : Optional Force execution even if the process is already ongoing
+	 *
 	 * ## EXAMPLES
 	 *     wp vip-search health validate-contents
 	 *
 	 * @subcommand validate-contents
 	 */
 	public function validate_contents( $args, $assoc_args ) {
-		$results = \Automattic\VIP\Search\Health::validate_index_posts_content(
+		$health = new \Automattic\VIP\Search\Health();
```

Comment:
Something I changed recently is the `Health()` class now requires a `Search` instance to be passed to the constructor (for DI), so we'll have to add that here.